File: react/src/app/dto/LoginDto.ts
export default class RegistrationDto {

    private email: string;
    private password: string;
    
    constructor(email: string, password: string){
        this.email = email;
        this.password = password;
    }

    toJSON() {
        return {
            "email": this.email,
            "password": this.password
        }
    }

}

File: spring/game/src/main/java/com/gigilife/game/user/dto/LoginDto.java
package com.gigilife.game.user.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/*
 * DTO for login API (/api/user/login endpoint)
 */
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class LoginDto {

    private String email;

    private String password;

}
